sct.py

```diff
@@ -481,11 +482,12 @@ def list_resources(ctx, user, test_id, get_all, get_all_running, verbose):
               type=CloudRegion(cloud_provider="aws"),
               default='eu-west-1',
               help="a region to look for AMIs (default: eu-west-1)")
-def list_ami_versions(region):
+@click.option('-a', '--arch', type=click.Choice(['x86_64', 'arm64']))
```

Comment:
it should have had a default here (and IMHO for now, it should default to `x86_64`)